python/.gitignore

```diff
@@ -4,7 +4,8 @@
 __pycache__/
 *.py[cod]
 *$py.class
-
+*.json
+*Test.py
```

Comment:
These aren't regularly added in this repo so let's not add them here. If you want to create a .gitignore in speech-to-text for the google json file, please do that instead.